source/mapping-rules/create-rules.txt

```diff
@@ -0,0 +1,106 @@
+====================
+Create Mapping Rules
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
```

Comment:
[nit] For this, I think we want the `depth` value to be `1` so each step in the procedure doesn't show up on the in-page TOC as a header.